src/style.css

```diff
@@ -0,0 +1,148 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --body-bc-color: rgb(229, 229, 229);
+  --page-text-color: rgb(6, 11, 53);
+  --code-price-color: rgb(97, 96, 112);
+  --white-color: rgb(255, 255, 255);
+  --button-color: rgb(0, 172, 220);
+  --border-color: rgb(243, 243, 243);
+}
+
+html {
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+.body {
+  background-color: var(--body-bc-color);
+  border-radius: 5px;
+  font-weight: 400;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white-color);
+  justify-content: center;
+  align-items: center;
+  border: 1px solid var(--border-color);
+  border-radius: 5px;
+}
+
+.stars {
```

Comment:
move each bem block to separate file

Reply:
Now the `html corrected` commit is the fixed one (hope so))